src/constants/label.js

```diff
@@ -1,9 +1,24 @@
 "use strict";
 
 // Name fields in order of preference
-export const name_en = [
-  "coalesce",
-  ["get", "name:en"],
-  ["get", "name:latin"],
-  ["get", "name"],
-];
+export const localizedName = (function () {
+  let userLocales =
+    "languages" in navigator ? [...navigator.languages] : [navigator.language];
+  let locales = [];
+  let localeSet = new Set(); // avoid duplicates
+  for (let locale of userLocales) {
+    // Add progressively less specific variants of each user-specified locale.
+    let components = locale.split("-");
+    while (components.length > 0) {
+      let parent = components.join("-");
+      if (!localeSet.has(parent)) locales.push(parent);
+      localeSet.add(parent);
+      components.pop();
+    }
+  }
+  if (locales.at(-1) === "en") {
+    locales.push("latin");
```

Comment:
This Latin fallback is vestigial at this point, but there remains another Latin fallback for roads and such:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/830d43690aa2a8e2c2e14a63e79deba259ffd859/src/layer/transportation_label.js#L82-L87

I’d like to replace this with `Label.localizedLabel`, but I’m unsure what the expected behavior would be for street names. Do we really need to split apart the Latin and non-Latin characters in a name? Wouldn’t they already be separate in a typical bilingual `name`?